src/ciphersuite/mod.rs

```diff
@@ -121,7 +150,7 @@ pub struct Ciphersuite {
     hpke_kem: HpkeKemMode,
     hpke_kdf: HpkeKdfMode,
     hpke_aead: HpkeAeadMode,
-    aead: AeadMode,
+    aead_mode: AeadMode,
```

Comment:
I think we should have it for the other modes as well, but for now I've only refactored the AEAD part of `ciphersuite.rs`.

Reply:
In that case all of this should be done in one go. It's not good practice to do partial renaming without a good reason. Since it's not really related to this PR, I would recommend doing a dedicated PR.

Reply:
Done!